### ISSUE TITLE
Cannot Watch Movies with plexdrive and plex

### ISSUE THREAD

**Issue by TheFuzz4** [dweidenfeld/plexdrive#299]
I have mounted the plexdrive and plex can see all of the files just fine.  But every time I go to watch something it throws in plex an s1001 error.  I've researched this to the moon and back about permissions and this and that and I have everything as far as I can tell setup correctly.  I'm hoping that someone might have some insight into something that maybe they've experienced this before.  The folders I'm trying to watch from were shared with me by someone else in my gdrive I don't know if that plays into this or not.  Thanks.

**Comment by HazCod** [comment]
Is this still an issue?

**Comment by TheFuzz4** [comment]
Yeah I'm still having issues with it.  This is what I"m seeing in my logs
`Jun 22, 2018 10:13:09.287 [0x7f5b1abf6700] ERROR - MDE: video has neither a video stream nor an audio stream
Jun 22, 2018 10:13:09.287 [0x7f5b1abf6700] ERROR - Streaming Resource: Cannot make a decision because either the file is unplayable or the client provided bad data
`
I've attempted to have plex analyze everything in the libraries in order to find the audio/video stream.  Someday's it works just fine but its a lot of miss before a hit.

**Comment by felixbuenemann** [comment]
I think it was mentioned in another issue, that Google applies much more aggressive rate limiting on shared files.
